templates/zerver/documentation_main.html

```diff
@@ -18,7 +18,7 @@ <h1><a href="{{ doc_root }}" class="no-underline">{{ doc_root_title }} home</a><
             {% if page_is_policy_center %}
             {{ render_markdown_path(sidebar_index, pure_markdown=True) }}
             {% elif page_is_help_center %}
-            {{ render_markdown_path(sidebar_index, context=api_uri_context, help_center=True) }}
+            {{ render_markdown_path(sidebar_index, pure_markdown=True) }}
```

Comment:
I don't think we need to pass `context` or `help_center=True` for the sidebar since it doesn't have any include macros or help relative/settings links.